addons/website_forum/static/src/scss/website_forum.scss

```diff
@@ -226,6 +226,22 @@ $-forum-sidebar-width: map-get($container-max-widths, sm) / 2;
             outline: 0;
         }
     }
+
+    .o_wforum_sign_up_cta {
```

Comment:
As discussed, let's keep the base bootstrap behavior, no need to re-define it, even if it looks slightly off.